modules/rostests/apitests/user32/makeimg/readme.txt

```diff
@@ -0,0 +1,14 @@
+For reference see https://jira.reactos.org/browse/CORE-17005.
```

Comment:
```suggestion
For reference see https://jira.reactos.org/browse/CORE-17005
```
it is more safe without the dot if anybody would be willing to copy-paste the URL.